test2.py

```diff
@@ -10,4 +10,14 @@
 - False if the number is odd
 
 Test your function before committing to Github.
-"""
\ No newline at end of file
+"""
+
+
+def evenorodd(number):
+    if number % 2 ==0:
+        result = True
+    else:
```

Comment:
Nice code, you can also make it shorter by removing the if/else:
number % 2 == 0  gives True or False right away